### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by andrewgarvey**
Added 5 new resources 

I particularly like the one in the META section, Unsure if you would allow that to stay but it seemed too funny as a concept to not at least suggest it.

Cheers
Andrew

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -99,7 +99,7 @@ A curated list of resources for learning about natural language processing, text
 - [A Tidytext Analysis of the Weinstein Effect](https://www.gokhanciflikli.com/post/weinstein-effect/) Dec 2013.
 - [Comparison of the Most Useful Text Processing APIs](https://www.kdnuggets.com/2018/08/comparison-most-useful-text-processing-apis.html)
 - [100 Must-Read NLP Papers](https://github.com/mhagiwara/100-nlp-papers)
-
+- [Python Guide for dealing with Text Date](https://www.analyticsvidhya.com/blog/2018/02/the-different-methods-deal-text-data-predictive-python/)
```
Comment (stepthom) [inline README.md]: > Text Date

Typo?

File: README.md [hunk 2]
```diff
@@ -301,6 +302,8 @@ Errors in Data](https://journal.r-project.org/archive/2010-2/RJournal_2010-2_Sar
 - [Udemy: NLP - Natural Language Processing with Python](https://www.udemy.com/nlp-natural-language-processing-with-python/)
 - [Udemy: Deep Learning: Advanced NLP and RNNs](https://www.udemy.com/deep-learning-advanced-nlp/)
 - [Udemy: Natural Language Processing and Text Mining Without Coding](https://www.udemy.com/master-natural-language-processing-nlp-text-mining-without-coding/)
+- [Standford NLP Lecture Series](https://www.youtube.com/playlist?list=PL3FW7Lu3i5Jsnh1rnUwq_TcylNr7EkRe6)
```
Comment (stepthom) [inline README.md]: > Standford

Typo - should be Stanford

File: README.md [hunk 3]
```diff
@@ -301,6 +302,8 @@ Errors in Data](https://journal.r-project.org/archive/2010-2/RJournal_2010-2_Sar
...
+- [Python ML and AI Courses](https://pythonprogramming.net/)
```
Comment (stepthom) [inline README.md]: This is a nice resource, but I'd like to keep things focused on NLP specifically.

File: README.md [hunk 4]
```diff
@@ -459,6 +462,7 @@ Errors in Data](https://journal.r-project.org/archive/2010-2/RJournal_2010-2_Sar
 
 - [Deep Learning for NLP, advancements and trends in 2017](https://tryolabs.com/blog/2017/12/12/deep-learning-for-nlp-advancements-and-trends-in-2017/?utm_campaign=Data%2BElixir&utm_medium=email&utm_source=Data_Elixir_162)
 - [10 Exciting Ideas of 2018 in NLP](http://ruder.io/10-exciting-ideas-of-2018-in-nlp/)
+- [Repository of NLP Ressources](https://github.com/stepthom/text_mining_resources)
```
Comment (stepthom) [inline README.md]: I appreciate the humor! But let's save some poor soul a few hours of infinite looping.

File: README.md (hunk 1 above)
Comment (andrewgarvey) [reply README.md]: Yup, changed

File: README.md (hunk 2 above)
Comment (andrewgarvey) [reply README.md]: changed

File: README.md (hunk 3 above)
Comment (andrewgarvey) [reply README.md]: fair enough,
removed

File: README.md (hunk 4 above)
Comment (andrewgarvey) [reply README.md]: expected, but disappointing none the less, 
removed

**Comment by andrewgarvey** [comment]
Made the changes on this branch(and committed). Hopefully that is sufficient.  

Cheers
Andrew

**Comment by stepthom** [comment]
@andrewgarvey Thanks! 👍

**Pull request merged**